.github/scripts/verifyActions.sh

```diff
@@ -8,16 +8,29 @@ declare -r GREEN='\033[0;32m'
 declare -r RED='\033[0;31m'
 declare -r NC='\033[0m'
 
+declare LIB_PATH
+LIB_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && cd ../../ && pwd)/node_modules/diff-so-fancy"
+
+# Expose diff-so-fancy to path
+PATH=$LIB_PATH:$PATH
+
 # Rebuild all the Github Actions
 printf '\nRebuilding GitHub Actions...\n'
 npm run gh-actions-build
 
 # Check for a diff
 printf '\nChecking for a diff...\n'
-if [[ $(git diff --exit-code) -eq 0 ]]; then
+git diff --exit-code | diff-so-fancy | less --tabs=4 -RFX
+
+# Runs git diff quietly to get the exit code
+declare EXIT_CODE
```

Comment:
Hmmm does the `exit-code` from the first `git diff` basically get swallowed up by the other piped commands? If so, I would just get rid of the `--exit-code` flag in the first `git diff` run. I'd be willing to bet that there's a clean way to propagate that exit code through the piped `diff-so-fancy` and `less` commands though, which would be preferable.

Reply:
Hi @roryabraham , I was going to reply you and David after finishing here. 
I totally agree with you, but after running many tests with many options (like trying to get from this line: git diff --exit-code | diff-so-fancy | less --tabs=4 -RFX) and also (if [[ EXIT_CODE -eq 0 ]]; ) this directly nothing gave the the first exist code of the pipeline. I tried for a few hours with all the things I could think of.

Reply:
While I understand that an extra variable is not ideal, that was the best I could do.

Reply:
Okay, I suck at using bash so I'm not sure I have a better suggestion. I'm not super worried about it, but maybe @bondydaa knows how to do this with some bash wizardry?